packages/front-end/hooks/useExperimentalRefreshMode.ts

```diff
@@ -0,0 +1,20 @@
+import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
+
+import { getIsExperimentIncludedInIncrementalRefresh } from "@/services/experiments";
+import { useDefinitions } from "@/services/DefinitionsContext";
+
+// This hook returns list of projects user has permission for along with any projects already associated with a resource (e.g. metric.projects)
+export default function useExperimentalRefreshMode(
```

Comment:
question: Do we have `Experimental` because we expect the RefreshMode to go away / remove this hook?

Otherwise I would vote to call it `useExperimentRefreshMode` or even `useExperimentPipelineMode` (regular vs incremental)?